examples/walkthrough/README.md

```diff
@@ -1,73 +1,149 @@
 
 See the [design document](../../DESIGN.md) for more details.
 
+
+### Managing a Pod
+
+If you have the source checked out, the following command (executed from the root of the project) will create the pod defined above. Otherwise download [the pod definition](pod-nginx.yaml) first.
+
+```sh
+kubectl create -f examples/walkthrough/pod-nginx.yaml
+```
+
+To list all pods:
+
+```sh
+kubectl get pods
+```
+
+To view the pod's http endpoint, first determine which IP the pod was given:
+
+```sh
+kubectl get pod nginx -o=template -t={{.status.podIP}}
+```
+
+Provided your pod IPs are accessible (depending on network setup), you should be able to access the pod's http endpoint in a browser or curl on port 80.
+
+```sh
+curl http://$(kubectl get pod nginx -o=template -t={{.status.podIP}})
+```
+
+To delete the pod by name:
+
+```sh
+kubectl delete pod nginx
+```
+
+
 ### Volumes
 
-Now that's great for a static web server, but what about persistent storage?  We know that the container file system only lives as long as the container does, so we need more persistent storage.  To do this, you also declare a ```volume``` as part of your pod, and mount it into a container:
+That's great for a simple static web server, but what about persistent storage?
+
+The container file system only lives as long as the container does. So if your app's state needs to survive reboots or crashes you'll need to configure some persistent storage.  To do so, declare a ```volume``` in the pod definition, and mount it into one or more of the pod containers:
+
+Added a volume:
+```
+  volumes:
+  - name: redis-persistent-storage
+    emptyDir: {}
+```
+
+Reference the volume in our container section:
+```
+    volumeMounts:
+    # name must match the volume name below
+    - name: redis-persistent-storage
+      # mount path within the container
+      mountPath: /data/redis
+```
+
+Full definition ([source](pod-redis.yaml)):
+
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
-  name: storage
+  name: redis
 spec:
   containers:
-    - name: redis
-      image: redis
-      volumeMounts:
-          # name must match the volume name below
-        - name: redis-persistent-storage
-          # mount path within the container
-          mountPath: /data/redis
-  volumes:
+  - name: redis
+    image: redis
+    volumeMounts:
     - name: redis-persistent-storage
-      emptyDir: {}
-```
-
-Ok, so what did we do? We added a volume to our pod:
-```
+      mountPath: /data/redis
   volumes:
-    - name: redis-persistent-storage
-      emptyDir: {}
-```
-
-And we added a reference to that volume to our container:
-```
-      volumeMounts:
-          # name must match the volume name below
-        - name: redis-persistent-storage
-          # mount path within the container
-          mountPath: /data/redis
+  - name: redis-persistent-storage
+    emptyDir: {}
 ```
 
```

Comment:
Sorry, the lines have moved due to edits but since the two statements about "emptydir" and "hostdir" provide additional information (vs. being actionable steps), im suggesting to insert "Tips:" to introduce the remaining portion of this section:

Tips:
In Kubernetes, ```emptyDir``` Volumes live for the lifespan of the Pod, which.....

If you want to mount a directory that already exists in the file systee.....

See.....